projects/crates.io/rx/package.yml

```diff
@@ -0,0 +1,19 @@
+distributable:
+  url: https://github.com/cloudhead/rx/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+provides:
+  - bin/rx
+
+versions:
+  github: cloudhead/rx/tags
+  strip: /v/
+
+build:
+  dependencies:
+    rust-lang.org/cargo: '*'
+  script:
+    cargo install --locked --path . --root {{prefix}}
+
+test:
+  rx -h
```

Comment:
Can we add a better test? -h doesn't always prove the binary will run.

Reply:
I tried
```yaml
test:
  dependencies:
    curl.se: '*'
  script: |
    rx -h
    curl http://www.libpng.org/pub/png/PngSuite/basn6a08.png >test.png
    rx -u $FIXTURE test.png
  fixture: |
    e test.png
    p/clear
    p/add #ff0000
    w test2.png
    q
```
but it a) opens a window, and b) won't quit from init. So, I think we might be stuck.